spiffworkflow-frontend/src/components/ProcessInstanceListTable.tsx

```diff
@@ -62,6 +62,7 @@ import {
 } from '../interfaces';
 import ProcessModelSearch from './ProcessModelSearch';
 import ProcessInstanceReportSearch from './ProcessInstanceReportSearch';
+import ProcessInstanceListDeleteReport from './ProcessInstanceListDeleteReport';
```

Comment:
neat, I kind of like naming the component based on the name of the thing using it.

Reply:
Took a note from `ProcessInstanceListSaveAsReport`, seemed to work.